whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/LirRipeMaintainedAttributesValidator.java

```diff
@@ -30,7 +30,8 @@
     private static final List<AttributeType> RIPE_NCC_MANAGED_ATTRIBUTES = ImmutableList.of(
             AttributeType.MNT_BY,
             AttributeType.ORG,
-            AttributeType.ORG_TYPE);
+            AttributeType.ORG_TYPE,
+            AttributeType.COUNTRY);
```

Comment:
This validation is already done in CountryRipeMaintainedValidator.java. If you add it here also, there may be duplicate error messages in the response. 

I don't mind if you want to move it here for consistency, but then please delete the other validator.